auth/src/lib.rs

```diff
@@ -45,14 +51,47 @@ impl Args for Options {
                 Long("active") => {
                     active = true;
                 }
+                Long("username") if username.is_none() => {
+                    let val = parser
+                        .value()?
+                        .to_str()
+                        .ok_or(anyhow::anyhow!(
+                            "invalid username specified with `--username`"
+                        ))?
+                        .to_owned();
+                    
+                    username = Some(val);
+                }
+                Long("password") if password.is_none() => {
```

Comment:
You could add `init` as an additional condition, so that you have to pass `--init` first, otherwise password and username don't make sense.